.circleci/config.yml

```diff
@@ -450,10 +473,34 @@ jobs:
           command: |
             cd server
             yarn test --coverage
-            cd ..
       - store_artifacts:
           path: server/coverage
 
+# Publish the android app
+  publish-android-app-appcenter:
+    executor:
+      <<: *android-build
+    parameters:
+      distribution-audience:
+        # Use this flag to control the app distrubution audience
+        type: string
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: "Publishing Android app to AppCenter"
+          command: |
+            npm install webpack
+            echo no | npm install -g appcenter-cli
+            appcenter login --token ${APP_CENTER_API_TOKEN}
+            appcenter distribute release \
+              -f app/android/app/build/outputs/apk/release/app-release.apk \
+              -g << parameters.distribution-audience >> \
+              --app nico-hokeyapp-dzyz/TwilioVoice-React-Native-Reference-App-1 \
```

Comment:
Can we put this value into the `vblocks-dogfood-reference-app` CircleCI context.

Reply:
That doesn't make sense, parameterw.distribution-audience is not a env variable, its a parameter to the command

Reply:
I agree with @mhuynh5757. Even though bad actors can't do anything if they don't have the token, these are still private values that only people with access should know about. My bad that I missed this when I was reviewing. Let's not hardcode them here. People pulling/forking the repository would also be very confused.

Reply:
I understand and am sympathetic with the concern for our list of env vars/contexts blowing up, but in the case of the Reference App, since we're planning on encouraging forking, the less hard-coded values the better.